docs/quickstarts/packet.md

```diff
@@ -141,126 +124,112 @@ ssh-add ~/.ssh/id_rsa
+lokomotive-demo-pool-1-worker-1    True     KubeletReady    kubelet is posting ready status    
 
 Success - cluster is healthy and nodes are ready!
 ```
 
-Use the generated `kubeconfig` file to access the Kubernetes cluster and list nodes.
+## Verification
+
+Use the generated `kubeconfig` file to access the cluster:
 
 ```console
-export KUBECONFIG=./lokomotive-assets/cluster-assets/auth/kubeconfig
+export KUBECONFIG=$(pwd)/assets/cluster-assets/auth/kubeconfig
 kubectl get nodes
 ```
 
+Sample output:
+
+```console
+NAME                            STATUS   ROLES    AGE   VERSION
+lokomotive-demo-controller-0      Ready    <none>   33m   v1.17.4
+lokomotive-demo-pool-1-worker-0   Ready    <none>   33m   v1.17.4
+lokomotive-demo-pool-1-worker-1   Ready    <none>   33m   v1.17.4
+```
+
 ## Using the cluster
 
-At this point you have access to the Kubernetes cluster and can use it!
-If you don't have Kubernetes experience you can check out the [Kubernetes
-Basics official
-documentation](https://kubernetes.io/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro/)
-to learn about its usage.
+At this point you should have access to a Lokomotive cluster and can use it to deploy applications.
 
-**Note**: Lokomotive sets up a pretty restrictive Pod Security Policy that
-disallows running containers as root by default, check the [Pod Security Policy
-documentation](../concepts/securing-lokomotive-cluster.md#cluster-wide-pod-security-policy)
-for more details.
+If you don't have any Kubernetes experience, you can check out the [Kubernetes
+Basics](https://kubernetes.io/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro/) tutorial.
+
+>NOTE: Lokomotive uses a relatively restrictive Pod Security Policy by default. This policy
+>disallows running containers as root. Refer to the
+>[Pod Security Policy documentation](../concepts/securing-lokomotive-cluster.md#cluster-wide-pod-security-policy)
+>for more details.
 
 ## Cleanup
 
-To destroy the Lokomotive cluster, execute the following command:
+To destroy the cluster, execute the following command:
 
 ```console
 lokoctl cluster destroy --confirm
 ```
 
-You can safely delete the working directory created for this quickstart guide if you no longer
-require it.
+You can now safely delete the directory created for this guide if you no longer need it.
 
 ## Troubleshooting
 
-### Stuck at copy controller secrets
+In case the deployment fails, adding the `-v` flag to `lokoctl cluster apply` can provide useful
+information.
 
-If there is an execution error or no progress beyond the output provided below:
+### Stuck at "copy controller secrets"
 
 ```console
 ...
-module.packet-mycluster.null_resource.copy-controller-secrets: Still creating... (8m30s elapsed)
-module.packet-mycluster.null_resource.copy-controller-secrets: Still creating... (8m40s elapsed)
+module.packet-lokomotive-demo.null_resource.copy-controller-secrets: Still creating... (8m30s elapsed)
```

Comment:
As a user, how do I know if I'm in this case or not? IOW, how do I know if it's stuck or not? I need to know in which case I am to act accordingly.

Creating a cluster can take up to 15 minutes, easily. Maybe we should mention to enter the packet UI and if servers are being created, just wait more time? Or to wait at least 10-15 minutes?

Is very difficult to know if you are here, as it only possible to see the mentioned output if you run with `-v`. But we are not running with `-v` the first time. And if you just ctrl-c, and run again with `-v`, I think it will just duplicate servers if some server was being created but creation was not complete (takes ~10 min from creating to creation complete).

Therefore, if you ctrl-c and run with `-v` when servers were still being created, it will create a not very good experience,  as I said, will create servers that were not completed again, you will see them duplicated on packet UI. And even worse, I think a `lokomotive cluster destroy` will not destroy all the servers, in that case (as terraform is ony aware of the last created ones).

Reply:
Ideally? We should print an error telling the user what went wrong. In the context of this PR though, what are you suggesting I change?

I guess I was assuming that when something breaks the user would head to the "Troubleshooting" section, read the first line saying "use `-v`", then look for their symptom.

Reply:
It seems my suggestions as questions were not clear. My suggestion was:
> Maybe we should mention to enter the packet UI and if servers are being created

I was also pointing out other problems a user might face. It's okay if we don't want to fix them, as long as we are aware they can happen and it was intentional not to say anything about them.

Reply:
Would including `-v` with `lokoctl cluster apply` in this guide address your concerns here? I'm leaning in this direction as long as https://github.com/kinvolk/lokomotive/issues/564 isn't addressed.

Reply:
Yes, definitely. Pointed the same in the other comment :joy:!